scripts/variantstore/tieout/GIAB_TIEOUT.md

```diff
@@ -0,0 +1,174 @@
+# NOTES
+ - Previously there was an attempt to tie out the feature extract input to VQSR between WARP and GVS.  However, that was abandoned in favor of the truth-sample based comparison here due to non-determinism and bugs in GenomicsDB that were hard to compensate for.  However, the docs and scripts for doing this are in the branch `gvs_vqsr_input_tieout` under `scripts/variantstore/tieout/feature_extract`
+ 
+## Prerequisites
+
+ - rtg
+ - bcftools
+ - tabix
+ - python 3.7+
+
+## One time tasks
+```
+
+# create SDF for reference (used by rtg eval)
+# use local path to GRC38 reference fasta
+REFERENCE="/Users/kcibul/projects/references/hg38/v0/Homo_sapiens_assembly38.fasta"
+rtg format --output human_REF_SDF $REFERENCE
+
+# copy down truth data locally
+mkdir -p truth && gsutil -m cp gs://broad-dsp-spec-ops/gvs/truth/* truth/
+```
+
+## Obtain Truth sample VCFs 
+
+First, create a full cohort extract (as described in README.md) using the desired filter_set_name.  Assuming this is in a single gathered VCF of `gvs.vcf.gz`
+
+```
+# subset to chr20
+bcftools view -O z gvs.vcf.gz chr20 > gvs.chr20.vcf.gz
+tabix gvs.chr20.vcf.gz
+
+# extract each of the samples
+INPUT_VCF=gvs.chr20.vcf.gz
+
+gatk SelectVariants -V ${INPUT_VCF} --sample-name SM-G947Y --select-type-to-exclude NO_VARIATION -O NA12878.gvs.chr20.vcf.gz
```

Comment:
this may be minor, but can we document explicitly what `--select-type-to-exclude NO_VARIATION` does? i'm guessing it means only return sites at which this sample has a variant, i.e. exclude ref sites?

Reply:
👍